src/app/onion/dashboard/components/list/list.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="dashboard-list">
+  <div class="dashboard-list__top">
+    <div class="dashboard-list__title">
+      {{ title || 'Learning Objects' }} 
+      <span *ngIf="learningObjects">({{ learningObjects.length }})</span>
+    </div>
+    <div *ngIf="showOptions; else buttonPlaceholderTemplate" class="dashboard-list__options">
+      <div class="btn-group to-right">
+        <button class="button good">New <i class="far fa-plus"></i></button>
+        <button class="button neutral">Filter <i class="far fa-angle-down"></i></button>
+      </div>
+    </div>
+    <ng-template #buttonPlaceholderTemplate>
+      <div style="height: 40px;"></div>
+    </ng-template>
+  </div>
+  <div class="dashboard-list__table">
```

Comment:
Aria table identification please. Here is the link, https://www.w3.org/TR/wai-aria-practices/examples/table/table.html